### PR TITLE
[#3537] Item-Selector with "file upload" and "file browse" data sources fail on add action

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/3537 - [studio-ui] Item-Selector with File Upload and File Browse data sources fail on add action #3537

### CODE REVIEW

File: static-assets/components/cstudio-forms/data-sources/file-browse-repo.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ YAHOO.extend(CStudioForms.Datasources.FileBrowseRepo, CStudioForms.CStudioFormDa
 		var langBundle = CMgs.getBundle("contentTypes", CStudioAuthoringContext.lang);
 
 		var _self = this;
+		console.log('here');
```
Comment (rart) [inline static-assets/components/cstudio-forms/data-sources/file-browse-repo.js]: X

File: static-assets/components/cstudio-browse/browse.js [hunk 2]
```diff
@@ -535,7 +537,7 @@
                 currentResults.unshift(results.item);
                 results = currentResults;
 
-                var pathLabel = path.replace(/\//g, ' / ');
+                var pathLabel = path ? path.replace(/\//g, ' / ') : "";
```
Comment (rart) [inline static-assets/components/cstudio-browse/browse.js]: single quote

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/components/cstudio-forms/data-sources/file-browse-repo.js (hunk 1 above)
Comment (vestrada) [reply static-assets/components/cstudio-forms/data-sources/file-browse-repo.js]: Done

File: static-assets/components/cstudio-browse/browse.js (hunk 2 above)
Comment (vestrada) [reply static-assets/components/cstudio-browse/browse.js]: Done

**Review by rart — APPROVED** [review]

**Pull request merged**